shell-ui/src/index.js

```diff
@@ -16,8 +16,8 @@ import {defaultTheme} from '@scality/core-ui/dist/style/theme';
 const EVENTS_PREFIX = 'solutions-navbar--';
 export const AUTHENTICATED_EVENT: string = EVENTS_PREFIX + 'authenticated';
 
-export type TranslationAndGroups = { en: string, fr: string, groups?: string[], activeIfMatches?: string };
-export type MenuItems = {[path: string]: TranslationAndGroups }
+export type PathDescription = { en: string, fr: string, groups?: string[], activeIfMatches?: string, icon?: string, isExternal?: boolean };
```

Comment:
This commit (0edb071) should reference the ticket #3219 and mark it as closed, I guess.